smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -175,11 +175,6 @@ public async Task Configure(DeviceProvisioningMethod method, string image, strin
                 args += method.ManualConnectionString.Map(
                     cs => { return $" -Manual -DeviceConnectionString '{cs}'"; }).GetOrElse(string.Empty);
 
-                foreach (RegistryCredentials c in this.credentials)
-                {
-                    args += $" -Username '{c.User}' -Password (ConvertTo-SecureString '{c.Password}' -AsPlainText -Force)";
-                }
-
```

Comment:
We don't need to define the image manually if we remove this information.These credentials are only needed for edge agent image pull